Example/FlagsmithClient/AppDelegate.swift

```diff
@@ -48,6 +52,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     Flagsmith.shared.hasFeatureFlag(withID: "freeze_delinquent_accounts") { (result) in
       print(result)
     }
+    
+    // Try getting the feature flags concurrently
```

Comment:
Because this was the cause of the original problem the customer had that lead to https://github.com/Flagsmith/flagsmith-ios-client/pull/40#issuecomment-1814196352. By replicating the scenario in the example app I'm validating that this PR is working fine.

This is also covered in the unit tests.